properties.schema

```diff
@@ -21,11 +21,20 @@
                   "inputType": "Checkbox",
                   "validators": []
                 },
+                "_showAsPopup": {
+                  "type": "boolean",
+                  "required": false,
+                  "default": false,
+                  "title": "Show as popup",
+                  "help": "When set to `true`, the instruction error is shown in a notify popup. The default is `false`.",
+                  "inputType": "Checkbox",
+                  "validators": []
+                },
                 "title": {
                   "type": "string",
                   "required": false,
                   "default": "Instructions",
-                  "title": "Notify Title",
+                  "title": "Title",
                   "inputType": "Text",
                   "validators": [],
                   "help": "Set the notify title for the instruction error"
```

Comment:
Should this be prefixed with: When using the 'show as popup' option,

Reply:
Good catch. Fixed.